vcredist-aio@0.102.0: download via aria2 failed

vcredist-aio: 0.101.0 -> 0.102.0
Updating one outdated app:
Updating 'vcredist-aio' (0.101.0 -> 0.102.0)
Downloading new version
Starting download with aria2 ...
Download: 11/14 10:43:00 [ERROR] CUID#8 - Download aborted. URI=https://github.com/abbodi1406/vcredist/releases/download/v0.102.0/VisualCppRedist_AIO_x86_x64_10Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/abbodi1406/vcredist/releases/download/v0.102.0/VisualCppRedist_AIO_x86_x64_102.Download: Download Results:                                           ot found
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: e45fa7|ERR |       0B/s|C:/Users/fuad/scoop/cache/vcredist-aio#0.102.0#3e1bcc6.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

WARN  Download failed! (Error 3) Resource was not found
WARN  https://github.com/abbodi1406/vcredist/releases/download/v0.102.0/VisualCppRedist_AIO_x86_x64_102.zip
    referer=https://github.com/abbodi1406/vcredist/releases/download/v0.102.0/
    dir=C:\Users\fuad\scoop\cache
    out=vcredist-aio#0.102.0#3e1bcc6.zip

WARN  & 'C:\Users\fuad\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\fuad\scoop\cache\vcredist-aio.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.5 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=13908 --continue --summary-interval=0 --auto-save-interval=1
WARN
Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=vcredist-aio%400.102.0%3a+download+via+aria2+failed
Fallback to default downloader ...
The remote server returned an error: (404) Not Found.
URL https://github.com/abbodi1406/vcredist/releases/download/v0.102.0/VisualCppRedist_AIO_x86_x64_102.zip is not valid

--- a/bucket/vcredist-aio.json
+++ b/bucket/vcredist-aio.json
@@ -11,7 +11,7 @@
     "hash": "3c188e4651bcc07df2bdb06d69585158797b1bafb91557b09217621628cd5b23",
     "installer": {
         "script": [
-            "if (-not (is_admin)) { error 'This package requires admin rights to install'; return }",
+            "if (-not (is_admin)) { error 'This package requires admin rights to install'; break }",
             "Start-Process -FilePath \"$dir\\$fname\" -ArgumentList @('/ai', '/gm2') -Verb RunAs -Wait"
         ]
     },

